scripts/xtensa-build-zephyr.py

```diff
@@ -32,24 +32,28 @@
 	{
 		"name": "apl",
 		"PLAT_CONFIG": "intel_adsp_cavs15",
+		"CONFIG_OVERLAY": "overlay.conf",
```

Comment:
This means that we won't support xtensa-build-zephyr.sh script anymore?

Cc: @marc-hb

Reply:
@dbaluta plan is to have one script and the python version is easier to maintain and develop (supports Windows too). Are you using the bash script internally or with yocto - if so we need to create a transition plan.

Reply:
@lgirdwood thanks for the info. We are fine with using the python script with next SOF release. Should be v2.1 right?

BTW, any schedule when that will be released?

Reply:
SOF stable-v2.1 has been branched, it's too late to deprecate xtensa-build-zephyr.sh in 2.1

xtensa-build-zephyr.sh should absolutely be deprecated for 2.2, much earlier in fact. It will take a bit of time to make sure  all CIs have switched but letting xtensa-build-zephyr.sh "bitrot" already  now is a good way to put pressure on CI.